testing/correctness/apps/window_detector/window_detector.py

```diff
@@ -158,7 +161,12 @@ def output(self, key, accumulator):
 @wallaroo.computation_multi(name="Split Accumulated")
 def split_accumulated(data):
     key, values, ts = data
-    return [(key, v, ts) for v in values]
+    if (values != tuple(sorted(values))):
```

Comment:
Was this meant to be removed?  If not, we should make it clear what's being printed